apps/vanilla/src/pages/2/logick/index.ts

```diff
@@ -0,0 +1,39 @@
+import { TurnGenerator } from './turnGenerator';
+import { DiceGenerator } from './diceGenerator';
+import { Player } from './player';
+import { ResultDisplay } from './resultDisplay';
+
+/** Game is class that collects all game components. */
+class Game {
+	/** Turn Generator. */
+	private turnGenerator: TurnGenerator;
```

Comment:
Don't forget about `readonly`.
```suggestion
	private readonly turnGenerator: TurnGenerator;
```